lib/mirage_block_partition.ml

```diff
@@ -13,19 +13,24 @@ module Make(B : Mirage_block.S) = struct
   type nonrec error = [ 
     | Mirage_block.error
     | `Block of B.error
-    | `Out_of_bounds ]
+    | `Out_of_bounds 
+    | `Not_connected ]
```

Comment:
`Mirage_block.error` defines a polymorphic variant `` `Disconnected``. Let's use that instead :-)